### PR TITLE
Update information-protection-deploy-assess.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about using AIP to learn about sensitive information types in on-premises data stores.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3478

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/solutions/information-protection-deploy-assess.md [hunk 1]
```diff
@@ -227,3 +227,8 @@ In addition to the Content Explorer, organizations have access to the Content Se
 Detailed guidance on the use of Content Search for discovery of personal data is provided in [this article](../compliance/search-for-and-find-personal-data.md). Content Search and other discovery techniques are also explored in [DSRs for the GDPR and CCPA](../compliance/gdpr-dsr-office365.md#introduction-to-dsrs).
 
 Additional insights on investigative and remediation techniques for personal data in Microsoft 365 are provided in the [monitor and respond article](information-protection-deploy-monitor-respond.md).
+
+> [!NOTE]
+> To Find what sensitive information you have in files stored on-premises, Please refer to [Azure Information Protection](https://docs.microsoft.com/azure/information-protection/quickstart-findsensitiveinfo)
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/information-protection-deploy-assess.md]:
```suggestion
> To Find what sensitive information you have in files stored on-premises, please refer to [Azure Information Protection](https://docs.microsoft.com/azure/information-protection/quickstart-findsensitiveinfo).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@BrendaCarter Please merge this PR. Thanks.

**Pull request merged**